frontend/src/pages/CreateEventPage.tsx

```diff
@@ -10,14 +10,33 @@ import useCreateRoom from '@/hooks/useCreateRoom';
 import Information from '@/components/Information';
 import { useTheme } from '@emotion/react';
 import IInfo from '@/icons/IInfo';
+import useSectionValidation from '@/hooks/CreateRoom/useSectionValidation';
 
 const CreateEventPage = () => {
   const navigate = useNavigate();
   const { colors } = useTheme();
-  const { title, availableDates, time, deadLine, isPublic, isReadyToCreateRoom, roomInfoSubmit } =
-    useCreateRoom();
+
+  const {
+    title,
+    availableDates,
+    time,
+    deadLine,
+    isPublic,
+    isCalendarReady,
+    isBasicReady,
+    roomInfoSubmit,
+  } = useCreateRoom();
+
+  const { showValidation, shouldShake, validateSection } = useSectionValidation(
+    isCalendarReady,
+    isBasicReady
+  );
 
   const handleCreateRoom = async () => {
+    if (!validateSection()) {
+      return;
+    }
+
     const session = await roomInfoSubmit();
     if (session) {
       navigate(`/check?id=${session}`);
```

Comment:
```suggestion
const handleCreateRoom = async () => {
    if (validateSection()) {
      const session = await roomInfoSubmit();
      if (session) {
      navigate(`/check?id=${session}`);
    }
   
```

이렇게 명시적으로 쓰는건 어떻게 생각하실까요? !로 처리하니까, 현재 지은 함수 네이밍이 조금 바로 와닿지 않았던 것 같아요!